src/routes/ConsortiumManager/settings/authorization-roles/AuthorizationRolesSettings.js

```diff
@@ -17,6 +18,7 @@ const AuthorizationRolesSettings = () => {
         <Route exact path={`${AUTHORIZATION_ROLES_ROUTE}/compare`} component={CapabilitiesCompare} />
         <Route exact path={`${AUTHORIZATION_ROLES_ROUTE}/create`} render={() => <RoleCreate path={AUTHORIZATION_ROLES_ROUTE} />} />
         <Route exact path={`${AUTHORIZATION_ROLES_ROUTE}/:id/edit`} render={() => <RoleEdit path={AUTHORIZATION_ROLES_ROUTE} />} />
+        <Route exact path={`${AUTHORIZATION_ROLES_ROUTE}/compareUsers`} component={UsersCapabilitiesCompare} />
```

Comment:
Probably, the "kebab" case in the route paths would be more consistent to other places

Reply:
fixed